src/components/Search.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react'
 
-const Search = () => {
+const Search = (props) => {
   return (
     <div className="ui huge fluid icon input">
       <input
         type="text"
         placeholder={"Search your Recent Transactions"}
```

Comment:
don't forget to also bind the value of the input. We want this to be a controlled component.